src/main/java/taxi/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+@WebServlet ("/myCars")
```

Comment:
```suggestion
@WebServlet ("/drivers/cars")
```
also, let's use only annotations or only web.xml in order to configure your url mappings